helpers/freedom_helper.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const cudl = require('cuddle');
+const winston = require('winston');
+const config = require('config/config');
+const accounts = require('freedom-accounts-util');
+
+const { rejectUnauthorized } = config.CUDL;
+const {
+    base_url,
+    client_id,
+    user_agent,
+    endpoints,
+    client_secret
+} = config.FREEDOM;
+
+const DASHBOARD_SCOPES = config.ACCOUNTS_API.scopes.DASHBOARD;
+
+
+module.exports = {
+    get_user_information,
+    get_oauth_access_token
+};
+
+async function get_oauth_access_token(params) {
+    params.client_id = client_id;
+    params.client_secret = client_secret;
+
+    let access_token;
+
+    try {
+        ({access_token} = await accounts.generate_token(DASHBOARD_SCOPES.USER_READONLY));
```

Comment:
Why this is enclosed inside a pair of parenthesis?

Reply:
This will fetch the `access_token` and assign it to the variable `access_token`.

Reply:
@johnurbano12 this is something new 
```
  var chars = {
  'a':  1,
  'b':  2,
  'c':  3
};

({a} = chars);
console.log(a); // print 1
````

Reply:
This should not be enclosed in parenthesis. Don't declare then destructure.